src/script/common/c_content.cpp

```diff
@@ -1851,9 +1851,10 @@ void read_hud_element(lua_State *L, HudElement *elem)
 	elem->name    = getstringfield_default(L, 2, "name", "");
 	elem->text    = getstringfield_default(L, 2, "text", "");
 	elem->number  = getintfield_default(L, 2, "number", 0);
-	elem->item    = getintfield_default(L, 2, "item", 0);
 	if (elem->type == HUD_ELEM_WAYPOINT) {
 		elem->item = getintfield_default(L, 2, "item", -1) + 1;
```

Comment:
Should I rename this to `precision`?

Reply:
and btw, { } are not needed in this case. wWthout them, the code looks neater

Reply:
I know, but I prefer to use them.

Reply:
Update: Will remove the curly braces according to the preferred code style